projects/plugins/jetpack/_inc/client/constants/urls.js

```diff
@@ -3,3 +3,6 @@ import { getRedirectUrl } from '@automattic/jetpack-components';
 export const imagePath = window.Initial_State.pluginBaseUrl + '/images/';
 export const JETPACK_CONTACT_SUPPORT = getRedirectUrl( 'jetpack-contact-support' );
 export const JETPACK_CONTACT_BETA_SUPPORT = getRedirectUrl( 'jetpack-contact-support-beta-group' );
+
+export const GETTING_STARTED_WITH_JETPACK_BACKUP_VIDEO_URL =
+	'https://www.youtube.com/watch?v=pS5-i5KjKTw';
```

Comment:
Could this video be embedded into the page, and use a VideoPress player? It seems to go against our brand to use a competitor right there in the Jetpack dashboard, link to their website, instead of showing off our own player.

Reply:
Thanks for this remark! After the [discussion](p1655898249879569-slack-C0D96691V) with design, we've decided to change the youtube link with the link to the video section in our documentation:

https://jetpack.com/support/backup/the-jetpack-backup-plugin/getting-started-with-the-jetpack-backup-plugin/#starting-your-first-backup